pkg/util/logs/logs.go

```diff
@@ -4,41 +4,30 @@
 package logs
 
 import (
-	"flag"
-	"log"
-	"time"
+	"os"
+	"strconv"
 
-	"github.com/golang/glog"
-	"github.com/spf13/pflag"
-	"k8s.io/apimachinery/pkg/util/wait"
+	"github.com/rs/zerolog"
 )
 
-var logFlushFreq = pflag.Duration("log-flush-frequency", 5*time.Second, "Maximum number of seconds between log flushes")
-
-// GlogWriter serves as a bridge between the standard log package and the glog package.
-type GlogWriter struct{}
-
-// Write implements the io.Writer interface.
-func (writer GlogWriter) Write(data []byte) (n int, err error) {
-	glog.Info(string(data))
-	return len(data), nil
-}
-
-// InitLogs initializes logs the way we want for kubernetes.
+// InitLogs initializes logs with Time and Global Level of Logs set at Info
 func InitLogs() {
-	log.SetOutput(GlogWriter{})
-	log.SetFlags(0)
-	flag.Set("logtostderr", "true")
-	// The default glog flush interval is 30 seconds, which is frighteningly long.
-	go wait.Until(glog.Flush, *logFlushFreq, wait.NeverStop)
-}
-
-// FlushLogs flushes logs immediately.
-func FlushLogs() {
-	glog.Flush()
-}
-
-// NewLogger creates a new log.Logger which sends logs to glog.Info.
-func NewLogger(prefix string) *log.Logger {
-	return log.New(GlogWriter{}, prefix, 0)
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+
+	// Log levels are outlined as follows:
+	// Panic: 5
+	// Fatal: 4
+	// Error: 3
+	// Warn: 2
+	// Info: 1
+	// Debug: 0
+	// Trace: -1
+	// more info can be found at https://github.com/rs/zerolog#leveled-logging
+
+	envLog := os.Getenv("LOG_LEVEL")
+	if val, err := strconv.Atoi(envLog); envLog != "" && err == nil && val >= -1 && val <= 5 {
+		zerolog.SetGlobalLevel(zerolog.Level(val))
+	} else {
+		zerolog.SetGlobalLevel(zerolog.InfoLevel)
+	}
```

Comment:
Is it possible to set up the timestamp globally here (not just the format) instead of doing the With().Timestamp() everywhere?